### DISCUSSION TITLE
`gray` color isn't available anymore at tmux 3.5

### DISCUSSION THREAD

**Discussion by barraIhsan** [tmux/tmux#4142] [General]
`/home/barra/.config/tmux/tmux.conf:62: invalid style: fg=gray,bg=default`

However, I don't see any changes to color on the 3.5 changelog, anyone knows why?

**Comment by ghost** [comment]
You must be new to tmux. Welcome friend. The maintainer(s) do a great service to us all, but they also really love breaking configuration files. I've found that replacing gray with colour244 works.

**Reply by barraIhsan** [reply]
> but they also really love breaking configuration files

lmao

> colour244 works.

ahh so with the 256-color mode? okay, thanks!

**Reply by barraIhsan** [reply]
wait, https://github.com/tmux/tmux/commit/9bd039d1bc5f9109fb83fd2ed85e17d02908af19
lmao
